core/errors/packager.h

```diff
@@ -0,0 +1,19 @@
+#ifndef SORBET_CORE_ERRORS_PACKAGER_H
+#define SORBET_CORE_ERRORS_PACKAGER_H
+#include "core/Error.h"
+
+namespace sorbet::core::errors::Packager {
+constexpr ErrorClass MustBeTypedStrict{8001, StrictLevel::False};
+constexpr ErrorClass InvalidPackageDefinition{8002, StrictLevel::False};
+constexpr ErrorClass RedefinitionOfPackage{8003, StrictLevel::False};
+constexpr ErrorClass PackageNotFound{8004, StrictLevel::False};
+constexpr ErrorClass UnpackagedFile{8005, StrictLevel::False};
+constexpr ErrorClass InvalidImportOrExport{8006, StrictLevel::False};
+constexpr ErrorClass MultiplePackagesInOneFile{8007, StrictLevel::False};
+constexpr ErrorClass MultipleExportMethodsCalls{8008, StrictLevel::False};
+constexpr ErrorClass NoSelfImport{8009, StrictLevel::False};
+constexpr ErrorClass InvalidPackageExpression{8010, StrictLevel::False};
+constexpr ErrorClass PackageFileMustBeStrict{8011, StrictLevel::False};
+constexpr ErrorClass InvalidPackageName{8012, StrictLevel::False};
```

Comment:
It looks like all these errors are at `typed: false`. If there's an underlying reason why they have to be `typed: false`, what do you think about adding a comment to the top of the file saying why someone might want to second guess themself if they're trying to add a new error at `typed: true` or higher here?

Reply:
Hmm, maybe I should change this.

* All of these errors occur in `__package.rb` files.
* `__package.rb` files must be `# typed: strict` (otherwise, one of these errors fires)
* Perhaps all errors but the `# typed: strict` error should require true or higher?

Reply:
The strictness level is the level below which the error is silenced.

So put another way, my question was: is there any error that could be added here that could be silenced in any way other than ignoring the file completely? It seems like every `__package.rb` error should be un-ignorable.

Reply:
Comment added.